src/uwtools/drivers/forecast.py

```diff
@@ -142,78 +136,53 @@ def resources(self, platform: dict) -> Mapping:
         """
         Parses the config and returns a formatted dictionary for the batch script.
         """
-        # Add required fields to platform.
-        # Currently supporting only slurm scheduler.
-        return {
-            "account": platform["account"],
-            "nodes": 1,
-            "queue": "batch",
-            "scheduler": self.job_scheduler,
-            "tasks_per_node": 1,
-            "walltime": "00:01:00",
-        }
 
-    def run(self) -> None:
+        return self._config["jobinfo"].update({
+            "account": self._expt_config["user"]["account"],
+            "scheduler": self._expt_config["platform"]["scheduler"],
+        })
+
+    def run(self, cdate, cyc) -> None:
         """
         Runs FV3 either as a subprocess or by submitting a batch script.
         """
-        # Read in the config file.
-        forecast_config = self._config["forecast"]
-        platform_config = self._config["platform"]
-
         # Prepare directories.
-        run_directory = forecast_config["RUN_DIRECTORY"]
+        run_directory = self._config["run_dir"]
         self.create_directory_structure(run_directory, "delete")
 
-        static_files = forecast_config["STATIC"]
-        self.stage_files(run_directory, static_files, link_files=True)
-        cycledep_files = forecast_config["CYCLEDEP"]
-        self.stage_files(run_directory, cycledep_files, link_files=True)
-
-        # Create the job script.
-        platform_resources = self.resources(platform_config)
-        batch_script = self.batch_script(platform_resources)
-        args = "--export=None"
-        run_command = self.run_cmd(
-            args,
-            run_cmd=platform_config["MPICMD"],
-            exec_name=forecast_config["EXEC_NAME"],
-        )
-        batch_script.append(run_command)
+        self._config["cycledep"].update(self._define_boundary_files())
```

Comment:
More of a philosophy/design comment but: Here, too, I feel like we could be more inclusive by being more explicit (is `dep` "dependent", "dependency", "deprecated", "depleted", ...) and avoiding jargon / tribal knowledge. It wouldn't be hard to write `cycle-dependent`, and probably people will be filling in blanks in a YAML template or tweaking previously used YAML files, so I think there's not much cost in being more explicit/readable. Just for your consideration...

Using shorter versions in code for conciseness may make sense, but the UW YAML will be the interface to humans and IMO ought to be as readable as we can manage.